Dockerfile

```diff
@@ -1,14 +1,12 @@
 FROM alpine:3.6
 
-LABEL maintainer="petr.pliska@post.cz"
+LABEL maintainer="Shopsys"
```

Comment:
It seems from examples I've seen that maintainer should always be an e-mail address...

Do we need the label? I would just remove it until we agree on having it in all our images.

Reply:
Ye, lets delete it